patch-diff-report-tool/src/main/resources/header.template

```diff
@@ -91,23 +91,23 @@
 							<td id="filesBase" th:text=${statistics.fileNumBase}> fileNumBase </td>
 							<td id="totalBase" th:text=${statistics.totalNumBase}> totalNumBase </td>
 							<th:block th:each="name : ${statistics.severityNames}">
-								<td th:id="${name} + @{SeverityNumBase}" th:text="${statistics.severityNumBase.get(name) != null} ? ${statistics.severityNumBase.get(name)} : '0'"> infoNumBase </td>
+								<td th:id="${name} + @{SeverityNumBase}" th:text="${statistics.severityNumBase.get(name)} ?: '0'"> infoNumBase </td>
 							</th:block>
 						</tr>
 						<tr class="a">
 							<td>patch</td>
 							<td id="filesPatch" th:text=${statistics.fileNumPatch}> fileNumPatch </td>
-							<td id="totalPatch" th:text=${statistics.totalNumPatch}> totalNumPatch </td>
+							<td id="totalPatch" th:text="${statistics.totalNumPatch} + (${statistics.uniqueMessagesBase > 0 || statistics.uniqueMessagesPatch > 0} ? (' (' + (${statistics.uniqueMessagesBase > 0} ? |${statistics.uniqueMessagesBase} removed| + (${statistics.uniqueMessagesPatch > 0} ? ', ' : '') : '') + (${statistics.uniqueMessagesPatch > 0} ? |${statistics.uniqueMessagesPatch} added| : '') + ')') : '')"> totalNumPatch </td>
 							<th:block th:each="name : ${statistics.severityNames}">
-								<td th:id="${name} + @{SeverityNumPatch}" th:text="${statistics.severityNumPatch.get(name) != null} ? ${statistics.severityNumPatch.get(name)} : '0'"> infoNumBase </td>
+								<td th:id="${name} + @{SeverityNumPatch}" th:text="(${statistics.severityNumPatch.get(name)} ?: '0') + (${statistics.severityNumRemoved.get(name) || statistics.severityNumAdded.get(name)} ? (' (' + (${statistics.severityNumRemoved.get(name)} ? |${statistics.severityNumRemoved.get(name)} removed| + (${statistics.severityNumAdded.get(name)} ? ', ' : '') : '') + (${statistics.severityNumAdded.get(name)} ? |${statistics.severityNumAdded.get(name)} added| : '') + ')') : '')"> infoNumBase </td>
 							</th:block>
 						</tr>
 						<tr class="d">
-							<td>difference</td>
-							<td id="filesDiff" th:text=${statistics.fileNumDiff}> fileNumDiff </td>
-							<td id="totalDiff" th:text=${statistics.totalNumDiff}> totalNumDiff </td>
+							<td/>
```

Comment:
@romani What is your opinion on removing the word `difference`?
In issue some screenshots still had `difference`, some had `changes in violations`, and some had no text like this PR.

Reply:
it is from column `Report Id` , so it is ok to remove, as we do not have such report ID, it is fake.
image without such word - https://github.com/checkstyle/contribution/issues/362#issuecomment-478247541
image with word - https://github.com/checkstyle/contribution/issues/362#issue-424254119

Reply:
This actually was what we agreed on in the issue, yes.
Also, it is not necessary anymore, as every column states what the number means now.

Reply:
I am with removal